libr/core/cmd_anal.c

```diff
@@ -4276,6 +4282,13 @@ static void cmd_aespc(RCore *core, ut64 addr, int off) {
 	r_core_seek (core, oldoff, 1);
 }
 
+static const char _handler_no_name[] = "<no name>";
+static int _aeli_iter(dictkv* kv, void* ud) {
+	RAnalEsilInterrupt* interrupt = kv->u;
+	r_cons_printf ("%3x: %s\n", kv->k, interrupt->handler->name ? interrupt->handler->name : _handler_no_name);
+	return 0;
```

Comment:
shouldnt return 1?

Reply:
(or bool?)

Reply:
not sure, think not, i think the iter returns 1 when it found the element so it stops calling it to exit early